go/api/base_client.go

```diff
@@ -286,3 +286,16 @@ func (client *baseClient) GetDel(key string) (string, error) {
 
 	return handleStringOrNullResponse(result), nil
 }
+
+func (client *baseClient) Del(keys ...string) (int64, error) {
+	if client.coreClient == nil {
+		return 0, &ClosingError{"Del command failed. The client is closed."}
+	}
```

Comment:
This is a redundant check and can be removed from here as it's already being checked in ExecuteCommand function.